packtools/sps/libs/reqs.py

```diff
@@ -1,13 +1,23 @@
+from packtools.sps import exceptions
+
 import requests
 
 
 def requests_get_content(uri, timeout=10):
-    response = requests.get(uri, timeout=timeout)
-    if response.status_code == 200:
+    try:
+        response = requests.get(uri, timeout=timeout)
+        response.raise_for_status()
+    except requests.exceptions.HTTPError:
+        raise exceptions.SPSHTTPError(f'{response.status_code}')
```

Comment:
@rafaelpezzuto como a função `requests_get_content` é genérica, talvez possa levantar a exceção HTTPError.
Acho que só faz sentido o raise exceptions.SPSHTTPError(f'{response.status_code}') se for para dar mais informação para a função que for tratar o erro.

Reply:
@robertatakenaka Ok, concordo em detalhar um pouco mais. Vou incluir erros que costumam ocorrer mais (403, 404, 500, 502 e 503) e quando for oportuno, abarcamos mais erros.